src/la/local_system.hh

```diff
@@ -28,6 +28,8 @@ class LocalSystem
 public:
     typedef arma::uvec DofVec;
     
+    static constexpr double artificial_zero = std::numeric_limits<double>::min();
```

Comment:
What about 'almost_zero', thats more descriptive. Also add detailed explanation of the whole problem here together with suggestions where to use it.